rust/tw_ss58_address/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "tw_ss58_address"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+blake2 = "0.10.6"
```

Comment:
Please consider using `blake2` implementation from the `tw_hash` crate

Reply:
Related to the comment https://github.com/trustwallet/wallet-core/pull/3857#discussion_r1615933825